admin/tools/forms.py

```diff
@@ -0,0 +1,13 @@
+from flask_wtf import FlaskForm
+from wtforms import FileField, PasswordField
+from wtforms.validators import DataRequired
+
+
+class UnlockFileForm(FlaskForm):
+    file = FileField(
+        validators=[
```

Comment:
The file upload validator on the main site is more comprehensive e.g.:
`
        validators=[
            StrictRequiredIf('button_clicked', 'Upload file',
                             message='Select a PDF file smaller than 10MB',
                             validators=[
                                 MultiFileAllowed(['pdf'],
                                                  message='Select a PDF file smaller than 10MB'),
                                 fileSizeLimit(10)
                             ]),
        ]
    )
`

Reply:
Yeh good idea, I'll add in those :-)